### PR TITLE
Fix circuit source bug

### LINKED ISSUE #2970: Bug in assign_power_sinusoidal_excitation_to_ports and assign_current_sinusoidal_excitation_to_ports
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

Looking at the code, both `assign_power_sinusoidal_excitation_to_ports` and `assign_current_sinusoidal_excitation_to_ports` create a source of type `CurrentSin`.

Oddly, `assign_power_sinusoidal_excitation_to_ports` , `assign_current_sinusoidal_excitation_to_ports` and also `assign_voltage_sinusoidal_excitation_to_ports` they all create a Power Sinusoidal source.

### Steps To Reproduce

Debug the unit test `test_21_Circuit.py` and inspect
- `test_21_assign_voltage_sinusoidal_excitation_to_ports`  
- `test_22_assign_current_sinusoidal_excitation_to_ports` 
- `test_23_assign_power_sinusoidal_excitation_to_ports`
they all create a Power Sinusoidal excitation

image

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

aiohttp==3.8.3
aiosignal==1.2.0
alabaster==0.7.12
ansys-sphinx-theme==0.8.0
anyio==3.6.1
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.8
async-timeout==4.0.2
attrs==22.1.0
Babel==2.10.3
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.1
certifi==2022.9.24
cffi==1.15.1
cfgv==3.3.1
charset-normalizer==2.1.1
click==8.1.3
click-plugins==1.1.1
cligj==0.7.2
clr-loader==0.2.3
codecov==2.1.12
colorama==0.4.5
commonmark==0.9.1
contourpy==1.0.5
coverage==6.5.0
cycler==0.11.0
debugpy==1.6.3
decorator==5.1.1
defusedxml==0.7.1
distlib==0.3.6
docutils==0.19
entrypoints==0.4
et-xmlfile==1.1.0
exceptiongroup==1.1.1
execnet==1.9.0
executing==1.1.1
fastjsonschema==2.16.2
filelock==3.8.0
Fiona==1.9.2
fonttools==4.37.4
frozenlist==1.3.1
geopandas==0.12.2
identify==2.5.6
idna==3.4
imageio==2.24.0
imageio-ffmpeg==0.4.8
imagesize==1.4.1
iniconfig==1.1.1
ipykernel==6.16.0
ipython==8.8.0
ipython-genutils==0.2.0
ipywidgets==8.0.4
jedi==0.18.1
Jinja2==3.1.2
joblib==1.2.0
json5==0.9.10
jsonschema==4.16.0
jupyter-core==4.11.1
jupyter-server==1.21.0
jupyter_client==7.4.1
jupyterlab==3.5.2
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.3
jupyterlab_server==2.15.2
kiwisolver==1.4.4
livereload==2.6.3
MarkupSafe==2.1.1
matplotlib==3.7.1
matplotlib-inline==0.1.6
mistune==2.0.4
multidict==6.0.2
munch==2.5.0
nbclassic==0.4.5
nbclient==0.7.0
nbconvert==7.2.1
nbformat==5.7.0
nbsphinx==0.8.12
nest-asyncio==1.5.6
networkx==3.0
nodeenv==1.7.0
notebook==6.4.12
notebook-shim==0.1.0
numpy==1.24.1
numpydoc==1.5.0
openpyxl==3.0.10
osmnx==1.3.0
packaging==21.3
pandas==1.5.3
pandocfilters==1.5.0
parso==0.8.3
pickleshare==0.7.5
Pillow==9.2.0
platformdirs==2.5.2
pluggy==1.0.0
plumbum==1.8.0
pooch==1.7.0
pre-commit==2.20.0
prometheus-client==0.14.1
prompt-toolkit==3.0.31
psutil==5.9.2
pure-eval==0.2.2
py==1.11.0
pycparser==2.21
pydata-sphinx-theme==0.9.0
Pygments==2.13.0
pypandoc==1.10
pyparsing==3.0.9
pyproj==3.4.1
pyrsistent==0.18.1
pytest==7.2.2
pytest-cov==4.0.0
pytest-forked==1.4.0
pytest-sphinx==0.5.0
pytest-xdist==3.2.0
python-dateutil==2.8.2
pythonnet==3.0.1
pytz==2022.4
pyvista==0.37.0
pywin32==303
pywinpty==2.0.8
PyYAML==6.0
pyzmq==24.0.1
recommonmark==0.7.1
requests==2.28.1
rpyc==5.3.0
scikit-learn==1.2.0
scikit-rf==0.25.0
scipy==1.9.2
scooby==0.6.0
Send2Trash==1.8.0
shapely==2.0.1
six==1.16.0
sklearn==0.0
sniffio==1.3.0
snowballstemmer==2.2.0
soupsieve==2.3.2.post1
Sphinx==5.3.0
sphinx-autobuild==2021.3.14
sphinx-copybutton==0.5.1
sphinx-gallery==0.11.1
sphinx-notfound-page==0.8.3
sphinx_autodoc_typehints==1.21.7
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
sphinxcontrib-websupport==1.2.4
SRTM.py==0.3.7
stack-data==0.5.1
terminado==0.16.0
threadpoolctl==3.1.0
tinycss2==1.1.1
toml==0.10.2
tomli==2.0.1
tornado==6.2
traitlets==5.4.0
urllib3==1.26.12
utm==0.7.0
virtualenv==20.16.5
vtk==9.2.2
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.4.1
widgetsnbextension==4.0.3
wslink==1.8.2
yarl==1.8.1

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Close #2970

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
good catch!
